trillian.proto

```diff
@@ -18,6 +18,7 @@ package trillian;
 
 import "github.com/google/trillian/crypto/sigpb/sigpb.proto";
 import "google/protobuf/any.proto";
+option java_outer_classname = "TrillianProto";
```

Comment:
Would you mind setting `option java_package = "com.google.trillian.proto";` here too?